### ISSUE TITLE
Default project type should be all the project types

### ISSUE THREAD

**Issue by jacksondaw** [cyotek/Cyotek.AddProjects#12]
Just like when you add an existing project to visual studio. The file type filter on the File Dialog should be all the project types. I believe the filter should also include the others, but I have found the current functionality inconvenient when working with a solution with the dreaded vb.net project.

**Comment by cyotek** [comment]
Hello,

That makes sense. I hadn't really noticed it as being a problem before as I mostly work with C# projects. When I do the VS2017 change I'll also add something that creates a combination _All Supported Projects_ entry which appears as the first item.

Thanks for the suggestion!

Regards;
Richard Moss

**Comment by cyotek** [comment]
Thanks for the PR. I forgot to mention that you could change in the configuration file; that would then allow you to have a combination filter without having to recompile the extension or wait for me to get off my behind and get it updated. I don't remember doing a UI for it so you'll have to tinker the file directly at `%AppData%\Cyotek\VisualStudioExtensions\AddProjects\config.xml`

**Comment by jacksondaw** [comment]
Yeah, I noticed that you added the ability to do it from the Settings as
well.
